WordPress/gutenberg
# Fix block gap in group blocks in classic themes

## What?

In preparation for WordPress 6.2, appearance-tools theme support was made available.

With the theme support, _but without theme.json_, the block gap setting does not work with the group block, because the CSS is only applied to the inner container div, the `wp-block-group__inner-container`, not the inner blocks.
Update: This affects **both** classic themes and block themes with the above condition.

This PR adds the `.wp-block-group__inner-container` as an additional partial selector for block gap.

Closes https://github.com/WordPress/gutenberg/issues/47386

## Why?

To allow classic themes to take advantage of new features.

## How?


## Testing Instructions
With WordPress 6.2 alpha:

1. Install and activate Twenty Twenty.
2. Add `add_theme_support( 'appearance-tools');` to the themes setup function in functions.php.
3. In the block editor, add one each of group, stack variation and row variation. (The latter two to confirm that nothing broke)
4. Set the block spacing setting to something visible, like 7.
5. Check if the block spacing is working. The editor and front should match.

## Screenshots or screencast
